### PR TITLE
Update supported Python version for kubeflow-katib SDK

### PR DESCRIPTION
**Pull request by tenzen-y**
As discussed in https://github.com/kubeflow/katib/pull/1793#pullrequestreview-862307507, we should update the supported Python version for our SDK.

/assign @andreyvelich @gaocegege @johnugeorge

### CODE REVIEW

File: sdk/python/v1beta1/setup.py [hunk 1]
```diff
@@ -38,8 +38,10 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
```
Comment (andreyvelich) [inline sdk/python/v1beta1/setup.py:40]: I think we can also remove these Python <= 3 versions, since they are not longer maintained.

**Review by andreyvelich — COMMENTED** [review]
Thank you for this update @tenzen-y!
We can also cherry pick this change on `release-0.13` branch.

File: sdk/python/v1beta1/setup.py (hunk 1 above)
Comment (tenzen-y) [reply sdk/python/v1beta1/setup.py]: Done.

**Comment by andreyvelich** [comment]
Thank you for the update @tenzen-y!
/lgtm
/approve

**Pull request merged**